### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by adamjedrzejec**

### CODE REVIEW

File: app/javascript/packs/Message/message_form.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      data.append('message[content]', this.state.content)
+      data.append('message[post_event_id]', this.state.postID)
+
+      axios.post("/messages.json", data, 
+      {headers: {
+          "X-CSRF-Token": document.querySelector('meta[name="csrf-token"]').content
+      }}).then(() =>
+        this.props.fetchMessages()
+      ).then(() =>
+        document.getElementById('textarea').value = ''
```
Comment (Vakrim) [inline app/javascript/packs/Message/message_form.jsx]: you shouldn't do any operations directly on DOM. Use react state to clear textarea.

File: app/javascript/packs/Message/message_form.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    render() {
+      return (
+        <>
+          <Form className="w-75">
+              <Form.Group controlId="exampleForm.ControlTextarea" className="mb-0">
+                <Form.Control id="textarea" as="textarea" placeholder="Wpisz komentarz..." rows="2" style={ this.state.errorDescription ? { borderColor: '#f23131'} : {} }
+                onChange={e =>{
+                  this.setState({content: e.target.value},
+                  this.validateContent())
+                }} />
```
Comment (Vakrim) [inline app/javascript/packs/Message/message_form.jsx]: add `value={this.state.content}` to have controlled component.

File: app/javascript/packs/Notification/notification.jsx [hunk 3]
```diff
@@ -227,7 +228,8 @@ export default class Notification extends React.Component {
                             </Row>                                                                          
                         </Modal.Body>
                     <Modal.Footer>
-                        <Col>Komentarze</Col>
+                        <Col><p>Komentarze</p>
+                        <MessageByNotification notifID={this.props.NotificationID} /></Col>
```
Comment (Vakrim) [inline app/javascript/packs/Notification/notification.jsx]: prefer using full names of variables/props
```suggestion
                        <MessageByNotification notificationID={this.props.NotificationID} /></Col>
```

File: app/assets/javascripts/messages.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (Vakrim) [inline app/assets/javascripts/messages.js]: remove unused files

File: app/assets/stylesheets/messages.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Messages controller here.
```
Comment (Vakrim) [inline app/assets/stylesheets/messages.scss]: remove unused files

File: app/controllers/messages_controller.rb [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    end
+
+    private
+
+    def set_post_event
+        @post_event = PostEvent.find(params[:id])
+    end
+
+    def message_params
+        params.require(:message).permit(:user_id, :content, :post_event_id)
```
Comment (hirokiraj) [inline app/controllers/messages_controller.rb]: remove `user_id`

File: app/controllers/messages_controller.rb [hunk 7]
```diff
@@ -0,0 +1,46 @@
+class MessagesController < ApplicationController
+    before_action :set_message, only: [:show, :edit, :update, :destroy]
+    before_action :authenticate_user!
+
+    def index
+        @messages = Message.all
+        # render json: @messages, status: :ok
+    end
+
+    def new
```
Comment (hirokiraj) [inline app/controllers/messages_controller.rb]: remove unused action

File: app/controllers/messages_controller.rb [hunk 8]
```diff
@@ -0,0 +1,46 @@
+class MessagesController < ApplicationController
```
Comment (hirokiraj) [inline app/controllers/messages_controller.rb]: shouldn't it be in `api/v1`?

File: app/views/messages/index.json.jbuilder [hunk 9]
```diff
@@ -0,0 +1,7 @@
+# json.array! @messages, partial: "messages/message", as: :message
```
Comment (hirokiraj) [inline app/views/messages/index.json.jbuilder]: if code is unused, then just remove it

File: app/views/messages/show_by_post_id.json.jbuilder [hunk 10]
```diff
@@ -0,0 +1,7 @@
+# json.array! @post_messages, partial: "post_messages/message", as: :message
```
Comment (hirokiraj) [inline app/views/messages/show_by_post_id.json.jbuilder]: if code is unused, then just remove it

File: app/javascript/packs/Message/message_form.jsx [hunk 11]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Form, Button } from 'react-bootstrap';
+import axios from 'axios';
+
+
+export default class CreateForm extends React.Component {
```
Comment (By698) [inline app/javascript/packs/Message/message_form.jsx]: Class name should be named as the file name but camelcase

File: app/javascript/packs/Notification/notification.jsx [hunk 12]
```diff
@@ -283,7 +284,8 @@ export default class Notification extends React.Component {
                             </Row>
                         </Modal.Body>
                     <Modal.Footer>
-                        <Col>Komentarze</Col>
+                        <Col><p>Komentarze</p>
+                        <MessageByNotification notificationID={this.props.NotificationID} /></Col>
```
Comment (By698) [inline app/javascript/packs/Notification/notification.jsx]: Correct the indentation

File: config/routes.rb [hunk 13]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  
+  root 'post_events#index'
 
   root 'post_events#index'
```
Comment (By698) [inline config/routes.rb]: Delete this repetition

**Pull request merged**